README.rst

```diff
@@ -45,16 +45,15 @@ Alternately, clone `the git repository`_ and execute `python setup.py install` w
 Configuration
 =============
 
-In your Django settings:
-
-- Add ``zesty_metrics`` to the ``INSTALLED_APPS``
-- Add ``zesty_metrics.middleware.MetricsMiddleware`` to ``MIDDLEWARE_CLASSES``
-- Set the following, as needed:
-  - ``STATSD_HOST``, default ``localhost``
-  - ``STATSD_PORT``, default ``8125``
-  - ``STATSD_PREFIX``, default ``None``
-  - ``ZESTY_TRACKING_CLASSES``, default ``('zesty_metrics.tracking.UserAccounts',)``
-- Run ``syncdb`` (or ``migrate`` if you use South).
+- In your Django settings:
+  - Add ``zesty_metrics`` to the ``INSTALLED_APPS``
```

Comment:
Is this correct? I thought there was some extended name that had to be in here now because of the app config?